mitodl/open-discussions
# Moderator UI for editing channel description

#### What are the relevant tickets?
Fixes #263 
Fixes #357 

#### What's this PR do?
Adds the UI to update the channel description for channel moderators

#### How should this be manually tested?
- Go to a channel page as a regular user, you should see no difference in UI from master
- Go to a channel page as a moderator, you should see an edit button above the channel description.
- Click the edit button and verify you can edit the channel description and the channel page updates to reflect that.
